### PR TITLE
update on tensorflow train example: object-detect's config file rewriting and file changing

### PR DESCRIPTION
**Pull request by FinchZHU**
The most recent release information of the Models/Research/Object-detect related to this change is as below.
Latest commit 7bf8934
Release information
April 30, 2018

We have released a Faster R-CNN detector with ResNet-101 feature extractor trained on [AVA](https://research.google.com/ava/) v2.1.
Compared with other commonly used object detectors, it changes the action classification loss function to per-class Sigmoid loss to handle boxes with multiple labels.
The model is trained on the training split of AVA v2.1 for 1.5M iterations, it achieves mean AP of 11.25% over 60 classes on the validation split of AVA v2.1.
For more details please refer to this [paper](https://arxiv.org/abs/1705.08421).

Thanks to contributors: Chen Sun, David Ross

Details:
Code for training within the newest version of Tensorflow is altered and no longer recognizes:

schedule {
            step: 0
            learning_rate: .0003
          }
Therefore it is removed.
Also, the newest create_pet_tf_record.py script creates batches of files in th form of pet_faces_val.record-00000-of-00010 and so on. The config file is changed accordingly on line 121 and line 135 to be the corresponding line.

The README is changed accordingly about the file changes and config rewriting.

### CODE REVIEW

**Comment by FinchZHU** [comment]
The base image should be uaitrain instead of uaiservice
